langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -379,9 +379,9 @@ public isolated function createText(string data) returns Text = @java:Method {
 #
 # ```ballerina
 # xml books = xml `<book>HTML</book><book>Invisible Man</book><book>David Copperfield</book><book>Jane Eyre</book>`;
-# books.slice(1, 3) ⇒ <book>Invisible Man</book><book>David Copperfield</book>
-# 
 # books.slice(2) ⇒ <book>David Copperfield</book><book>Jane Eyre</book>
+# 
+# books.slice(1, 3) ⇒ <book>Invisible Man</book><book>David Copperfield</book>
```

Comment:
suggestion:
Isn't it better if we add a example with `books.slice(0) => <book>HTML</book><book>Invisible Man</book><book>David Copperfield</book><book>Jane Eyre</book>`

Otherwise user may got confuse why we start the indexing from 1

Reply:
We need to show why we skip 0.
wdyt? @sanjana @MaryamZi @mohanvive